judge/models/tests/test_problem.py

```diff
@@ -245,6 +245,54 @@ def test_organization_admin_problem_methods(self):
         }
         self._test_object_methods_with_users(self.organization_admin_problem, data)
 
+    def give_basic_problem_ac(self, user):
+        Submission.objects.create(
+            user=user.profile,
+            problem=self.basic_problem,
+            result='AC',
+            points=self.basic_problem.points,
+            language=Language.get_python3(),
+        )
+
+    def test_problem_voting_permissions(self):
+        self.assertEqual(self.basic_problem.get_vote_perm(self.users['anonymous']), VotePermission.NONE)
+
+        _now = timezone.now()
```

Comment:
Nit, otherwise this implies being unused.

```suggestion
        now = timezone.now()
```